templates/configurations/form.interface.hbs

```diff
@@ -96,21 +96,23 @@
     </div>
 
     {{#isSlider}}
-    <div class="form-group cleafix">
-      <div class="col-xs-12">
-        <label>Minimum Value Number</label>
+    <div class="form-group cleafix" :class="{ 'has-error': _fieldSliderMaxMinError || _fieldSliderStepError}">
+      <div class="col-xs-12 slider-margin-bottom" >
```

Comment:
This is a terrible solution. Looking at the HTML markup, it's obvious that the problem with the padding occurs due to incorrectly usage of the Bootstrap framework or adherence to how we have used it in the rest of this file.

Every configuration field should be wrapped in `.form-group`, which provides the spacing between each configuration field. You've stuck 3 configurations into one `.form-group` here. Please read up on the rest of this file and follow our convention.

Reply:
done the changes